### DISCUSSION TITLE
ONLY USE DISCUSSIONS WITH LOW PRIORITY QUESTIONS/REQUESTS

### DISCUSSION THREAD

**Discussion by ClassicOldSong** [ClassicOldSong/moonlight-android#3] [Announcements]
Use issues if possible since discussions are not likely to notify users in time.

**Comment by OutPut-translate** [comment]
Hello. Are you planning to expand the list of supported platforms? Interested in webos? Is there any way to help without any development experience?

**Reply by ClassicOldSong** [reply]
You can try persuade the WebOS client maintainer to support Apollo ;-)

**Comment by JAMESON-42** [comment]
Hi, Can you help? I can't stream my pc on my Lenovo Xiaoxin Pad 2024 if the resolution is anything or a little bit higher than 1920x1080. Even native doesn't work and says my device doesn't support it? but its resolution is 1920x1200 
<img width="1920" height="1200" alt="Screenshot_20260703-214048" src="https://github.com/user-attachments/assets/1ab2ccbe-e903-4687-bbec-a5688c85d6a2" />

**Reply by ClassicOldSong** [reply]
Mid to low end SoC usually only supports decoding up to 1080P
